justfile

```diff
@@ -137,6 +137,16 @@ bench_against_main:
     cargo bench -p imap-codec-bench
     rm -rf target/bench_tmp
 
+# Build and test bindings
+bindings: bindings_python
+
+# Build and test Python bindings
+bindings_python: install_python_maturin
+    rm -rf target/wheels/*
```

Comment:
question: Can this fail when there is no `target` directory yet?

Reply:
I don't think so. From the `rm` man page: 
```
       -f, --force
              ignore nonexistent files and arguments, never prompt
```